blockbuster/lane.go

```diff
@@ -1,17 +1,21 @@
 package blockbuster
 
 import (
+	"github.com/cometbft/cometbft/libs/log"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkmempool "github.com/cosmos/cosmos-sdk/types/mempool"
-	"github.com/skip-mev/pob/mempool"
 )
 
 type (
-	// LaneConfig defines the configuration for a lane.
-	LaneConfig[C comparable] struct {
-		// XXX: For now we use the PriorityNonceMempoolConfig as the base config,
-		// which should be removed once Cosmos SDK v0.48 is released.
-		mempool.PriorityNonceMempoolConfig[C]
+	// LaneConfig defines the basic functionality needed for a lane.
+	LaneConfig struct {
+		Logger      log.Logger
+		TxEncoder   sdk.TxEncoder
+		TxDecoder   sdk.TxDecoder
+		AnteHandler sdk.AnteHandler
+
+		// Key defines the name of the lane.
+		Key string
```

Comment:
I don't see the purpose of a configurable key being passed in. The `Name()` should continue returning a constant IMO.

Reply:
good catch, i agree it shouldnt be configurable.